_widget/src/components/app/component.vue

```diff
@@ -73,7 +72,7 @@ export default {
 
   watch: {
     q (val) {
-      if (val.length > 2) {
+      if (val.length > 1) {
```

Comment:
Some record types are shorter than 2 characters. Sadly, A records are still left out.

Reply:
The reason for filtering these out is that a query of a single character is very difficult to determine intent. Something like `mx` has a clear intent.

Reply:
Maybe we could explicitly account for record types in the value? I.e. if the query value is "A", we show the A record articles. 

Related: currently in this branch, when searching `a record`, it shows the most relevant article last. Compared to the results in current production where the most relevant article is at the top.

| deploy-preview-1371--dnsimple-support.netlify.app | support.dnsimple.com |
|--------|--------|
| <img width="600" alt="image" src="https://github.com/user-attachments/assets/5ffd6438-059c-4a5a-af16-429a44255809" /> | <img width="550" alt="image" src="https://github.com/user-attachments/assets/711f6ab1-e908-4014-965d-ff019576f28e" /> |

Reply:
Hm... good find.

Reply:
Just tested and it's a lot better. Thanks!